lib/services/elasticsearch.js

```diff
@@ -1376,14 +1420,10 @@ class ElasticSearch extends Service {
           },
           toImport = [];
 
-        let idx = 0;
-
-        for (let i = 0; i < extractedDocuments.length; i++) {
-          const document = extractedDocuments[i];
-
+        while ((document = extractedDocuments.shift()) !== undefined) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "document" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS2703)